python/ray/experimental/state/common.py

```diff
@@ -696,6 +702,8 @@ def __post_init__(self):
 Summary API schema
 """
 
+DRIVER_TASK_ID_PREFIX = "ffffffffffffffffffffffffffffffffffffffff"
```

Comment:
Is there a better way to determine if a task_id refers to the driver?

Reply:
theoretically `TaskState.type` should indicate if it's a driver task. But we are currently filtering out drivers by default.

I am working on a PR to change that.

Reply:
Hmm not sure if it is also reliable. cc @rickyyx are we planning to have a PR to fix this? 

Maybe we can always include the driver task from the API, but we can filter it out when we print?

Reply:
One issue with that approach is if the list_tasks endpoint doesn't include the driver, we will drop the whole tree...

So going > 10k tasks has a very big chance of losing a lot of data.